pkg/database/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface Note {
+	_id: string;
+	books: BookInterface[];
+}
+
+export interface BookInterface {
+	isbn: string;
+	title: string;
+	author: string;
+	editor: string;
+	price: number;
+	quantity: number;
+	note?: string;
+	date?: string;
```

Comment:
This is a bit loose. We can create two interfaces: 
- `BookStock` (I believe that was the term @delasous suggested and we've agreed on it): This interface without the optional fields
- `BookStockInNote` in: Extends `BookStock` and adds required `note` and `date`

Reply:
After today's discussion we settled on `BookData` for title, author, etc. and `Volume`, when it also has date, quantity and an indication of where the book is (might be one of {`note`, `location`, `where`}).